ecpy_hqc_legacy/tasks/tasks/instr/run_awg.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Tasks to set the parameters of arbitrary waveform generators.
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+from atom.api import (Unicode, Bool, set_default, Enum)
+
+from ecpy.tasks.api import (InstrumentTask, InterfaceableTaskMixin,
+                            validators)
+import logging
```

Comment:
As per the docs the imports coming from the Python stdlib should come first so please move this import in first position.

Reply:
ok